spot_driver/test/test_stand.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2023 Boston Dynamics AI Institute LLC. See LICENSE file for more info.
```

Comment:
Copyright year is still wrong here on down :-p

Reply:
Not sure what happened, but the are all done now.